src/client/Microsoft.Identity.Client/AuthenticationResult.cs

```diff
@@ -263,5 +268,47 @@ public string CreateAuthorizationHeader()
                 _authenticationScheme?.AuthorizationHeaderPrefix ?? TokenType,
                 AccessToken);
         }
+
+#if !NETSTANDARD1_3
+        /// <summary>
+        /// Creates an object to hold the claims from an ID Token.
```

Comment:
`Returns a <See ClaimsPrincipal/> containing the claims extracted from the ID Token.`